graspologic/layouts/colors.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import atexit
+from itertools import cycle
+import json
+import math
+import numpy as np
+import os
+from pathlib import Path
+import pkg_resources
+from sklearn.preprocessing import minmax_scale
+from typing import Any, Dict, Optional, Tuple
+
+
+__all__ = ["nominal_colors", "sequential_colors"]
+
+
+def _load_thematic_json(path: Optional[str]) -> Tuple[Dict[Any, Any], Dict[Any, Any]]:
+    if path is not None and Path(path).is_file():
+        colors_path = path
+    else:
+        atexit.register(pkg_resources.cleanup_resources)
+        include_path = pkg_resources.resource_filename(__package__, "include")
+        colors_path = os.path.join(include_path, "colors-100.json")
+
+    with open(colors_path) as thematic_json_io:
+        thematic_json = json.load(thematic_json_io)
+    light = thematic_json["light"]
+    dark = thematic_json["dark"]
+    return light, dark
+
+
+_CACHED_LIGHT, _CACHED_DARK = _load_thematic_json(None)
+
+
+def _get_colors(light_background: bool, theme_path: Optional[str]) -> Dict[Any, Any]:
+    (
+        light,
+        dark,
+    ) = _CACHED_LIGHT, _CACHED_DARK if theme_path is None else _load_thematic_json(
+        theme_path
+    )
+    return light if light_background else dark
+
+
+def nominal_colors(
```

Comment:
@natoverse why do you call these "nominal" instead of "categorical" or "qualitative".

@bdpedigo I used nominal simply because that's what the json file that gives us our palette calls it.  And that's all on Nathan.

Reply:
We use nominal more or less interchangeably with categorical. Sometimes people lump in ordinal with categorical, so it seems a bit more precise to use "nominal" to refer to plain labeled data like this, but honestly I don't think it matters much. The underlying thematic library uses "nominal", but feel free to use categorical when you map it in graspologic if it aligns better with your ecosystem :).